tests/integration/conftest.py

```diff
@@ -6,7 +6,7 @@
 import pytest
 import yaml
 
-import utils
+from . import utils
```

Comment:
Just curious, why is importing like this better?

Reply:
It didn't work for me without it. python was not able to find module without installing cachito. Why cachito should be installed for int tests?